spec.html

```diff
@@ -26024,6 +26140,17 @@ <h1>Abstract Module Records</h1>
                 Purpose
               </th>
             </tr>
+            <tr>
+              <td>
+                GetModuleSource()
+              </td>
+              <td>
+                <p>It returns either a normal completion completion for the ModuleSource Object corresponding to this source Module Record's source phase (<emu-xref href="#sec-module-source-objects"></emu-xref>), or a throw completion.</p>
```

Comment:
Below this is spelled "Module Source Object", rather than "ModuleSource Object". It should be consistent (and ideally `<dfn>`'d).

Reply:
Added a space in https://github.com/tc39/proposal-source-phase-imports/pull/52. I don't want to call this a definition since these are a class of objects as opposed to a specific object here.